src/pages/components/toggle/style.mdx

```diff
@@ -72,34 +75,34 @@ and read-only states.
 <Row>
 <Column colLg={8}>
 
-![Interactive toggle states](images/toggle-style-states.png)
+![Example of default and small toggle interactive state color](images/toggle-style-states.png)
```

Comment:
Small thing, but in this interactive state color image, the labels for the focus state should say "Focus" and not "Focused", just to match the table terminology.